src/ResultList.php

```diff
@@ -116,15 +129,15 @@ public function getResults()
     }
 
     /**
-     * @return \ArrayIterator|\Traversable
+     * @return ArrayIterator|Traversable
      */
     public function getIterator()
```

Comment:
Is this used anywhere? I just grepped a v1.3.1 codebase, and I only see the method declaration itself.

Reply:
It's overriding base code  (SS_List?) I think, it's intended to provide the correct iterator logic to user code.